### PR TITLE
fix: sanitize markup

### LINKED ISSUE #75: Escape user content to prevent XSS in realworld example
The article body isn't correctly sanitized, example: https://realworld.svelte.dev/article/i-dd-dk6idd

### PR DESCRIPTION
**Pull request by wackbyte**
Closes #75.

### CODE REVIEW

**Comment by wackbyte** [comment]
Hm, not sure why it's failing. But other than that, would a library other than DOMPurify be preferred? (e.g. insane or sanitize-html)

**Comment by benmccann** [comment]
If we can avoid jsdom that would be nice

**Comment by wackbyte** [comment]
Changed to use sanitize-html

File: src/routes/article/[slug]/+page.server.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as api from '$lib/api.js';
 import { error, redirect } from '@sveltejs/kit';
 import { marked } from 'marked';
+import sanitizeHtml from "sanitize-html";
```
Comment (benmccann) [inline src/routes/article/[slug]/+page.server.js:4]:
```suggestion
import sanitizeHtml from 'sanitize-html';
```

**Pull request merged**